firebase.json

```diff
@@ -29,7 +29,7 @@
 		"runtime": "nodejs16"
 	},
 	"storage": {
-		"rules": "storage.rules"
+		"rules": "firestore.rules"
```

Comment:
Is this change on purpose? Here we are talking about storage rules not firestore no?

Reply:
Yes, you're right. I totally mixed this up.